Makefile

```diff
@@ -727,6 +730,18 @@ openstack: openstack_prep operator_namespace ## installs the operator, also runs
 	$(eval $(call vars,$@,openstack))
 	oc apply -f ${OPERATOR_DIR}
 
+.PHONY: openstack_ga
+openstack_ga: export BRANCH=${OPENSTACK_STABLE_BRANCH}
+openstack_ga: export REPO_DIR=${OPERATOR_BASE_DIR}/openstack-operator
+openstack_ga: export REGISTRY=${OPENSTACK_REGISTRY}
+openstack_ga: export NAMESPACE=${OPENSTACK_REGISTRY_NAMESPACE}
+openstack_ga: openstack_repo operator_namespace ## installs the operator for a feature branch using the first available index image matching a commit.
```

Comment:
I'm not convinced we need this target. Why not just use OPENSTACK_IMG as the way to drive this. This allows us to have realistic targets when upgrading FROM and TO something. Realistic in the sense that we could use an index that contains actual bundles we used for a historical downstream release for example quay.io/dprince/openstack-operator-index:1.0.2 is what we released for 18.0.2. We obviously want to upgrade TO the latest HEAD of the upstream maintenance branches but still this could be driven by simply using OPENSTACK_IMG right?

Reply:
yes with what we discussed yesterday its no longer needed. I'll update or submt a new pr later today

Reply:
And for CI we could simply set the OPENSTACK_IMG being use as the baseline FROM target for upgrades in the .ci-operator.yaml file of the openstack-operator repo. This way we wouldn't have to change anything in openstack/release when modifying our baseline FROM target.